src/app/machines/views/MachineList/MachineList.test.tsx

```diff
@@ -51,158 +183,24 @@ describe("MachineList", () => {
-              status: TestStatusStatus.FAILED,
-            }),
-            distro_series: "xenial",
-            domain: modelRefFactory({
-              name: "example",
-            }),
-            extra_macs: [],
-            fqdn: "other.example",
-            hostname: "other",
-            ip_addresses: [],
-            memory: 6,
-            memory_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            network_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            osystem: "ubuntu",
-            owner: "user",
-            permissions: ["edit", "delete"],
-            physical_disk_count: 2,
-            pool: modelRefFactory(),
-            pxe_mac: "66:77:88:99:00:11",
-            spaces: [],
-            status: NodeStatus.RELEASING,
-            status_code: NodeStatusCode.RELEASING,
-            status_message: "",
-            storage: 16,
-            storage_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            testing_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            system_id: "def456",
-            zone: modelRefFactory(),
-          }),
-          machineFactory({
-            actions: [],
-            architecture: "amd64/generic",
-            cpu_count: 2,
-            cpu_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            distro_series: "xenial",
-            domain: modelRefFactory({
-              name: "example",
-            }),
-            extra_macs: [],
-            fqdn: "other.example",
-            hostname: "other",
-            ip_addresses: [],
-            memory: 6,
-            memory_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            network_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            osystem: "ubuntu",
-            owner: "user",
-            permissions: ["edit", "delete"],
-            physical_disk_count: 2,
-            pool: modelRefFactory(),
-            pxe_mac: "66:77:88:99:00:11",
-            spaces: [],
-            status: NodeStatus.RELEASING,
-            status_code: NodeStatusCode.DEPLOYED,
-            status_message: "",
-            storage: 16,
-            storage_test_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            testing_status: testStatusFactory({
-              status: TestStatusStatus.FAILED,
-            }),
-            system_id: "ghi789",
-            zone: modelRefFactory(),
-          }),
-        ],
+        },
       }),
     });
   });
 
   afterEach(() => {
     localStorage.clear();
-  });
-
-  it("displays a loading component if machines are loading", () => {
```

Comment:
I moved this test to `MachineListTable.test.tsx` file as that's the component where the spinner appears.